src/articles/ArticlePreview.js

```diff
@@ -10,17 +18,74 @@ export default function ArticleOverview({
   dontShowImage,
   hasExtension,
 }) {
+  const [isOpen, setIsOpen] = useState(false);
+
   let topics = article.topics.split(" ").filter((each) => each !== "");
   let difficulty = Math.round(article.metrics.difficulty * 100) / 10;
 
+  function handleClose() {
+    setIsOpen(false);
+  }
+
+  function handleOpen() {
+    setIsOpen(true);
+  }
+
   function titleLink(article) {
     let open_in_zeeguu = (
       <Link to={`/read/article?id=${article.id}`}>{article.title}</Link>
     );
     let open_externally = (
-      <a target="_blank" rel="noreferrer" href={article.url}>
-        {article.title}
-      </a>
+      //TODO: Refactor into separate components and add styling to the modal and article titles
+      //Code related to the new redirection notification modal starts here
+      <>
+        <Modal open={isOpen} onClose={handleClose}>
+          <>
+            <MyBox>
+              <div>
+                <h1>
+                  You’re now leaving to&nbsp;a&nbsp;third&nbsp;party&nbsp;site.
+                </h1>
+
+                <BodyContainer>
+                  <ol>
+                    <li>Enter the article's site</li>
```

Comment:
Are the bullets needed? 
bullet 1 is done automatically - they don't have to do anything. 
bullet 2 is the thing that you want to tell them to actually do.
bullet 3 is implicit.

So in theory the bullets could be replaced with a sentence that reminds them to click on the extension to read with zeeguu. Something that continues the title, maybe something like: 

    Once there, click the Zeeguu Reader extension button in the top right corner of your browser. Then select Read.

Reply:
Something in this direction maybe? 

<img width="637" alt="image" src="https://github.com/zeeguu/web/assets/464519/ccfc8f52-6aab-4f0d-b39f-fc748a5a2792">

Reply:
I like the suggestion. Concise and to the point. 
I am also considering adding a brief instruction for those users who forgot to pin the extension and need more help finding it. The said instruction could be placed underneath the image, guiding the user on where to click to expand the extensions list.